tv_apps/smart-home/js/edit.js

```diff
@@ -352,7 +352,11 @@
     },
 
     deleteCard: function(scrollable, nodeElem) {
-      document.activeElement.blur();
+      // we have to move focus to another element to prevent user press enter
+      // to trigger unexpected behavior. We cannot use blur() to do it because
+      // it makes smart-home without focus. In this case, other apps can
+      // request focus from smart-home which makes focus lost, bug 1161940.
+      document.getElementById('main-section').focus();
```

Comment:
No, this is not enough.

This issue is not only related to `contextmenu`. It's related to hold a focus. If the app under smart-home make a timeout and request focus at the moment we show this dialog, it can grab the focus.